nssp/delphi_nssp/constants.py

```diff
@@ -0,0 +1,29 @@
+"""Registry for variations."""
+
+GEOS = [
+    "nation",
+    "state",
+    "county",
```

Comment:
suggestion: since we have county data available, there are various other geos we can easily aggregate up to (both MSAs and HRR are constructed along county boundaries). We should default to providing these for all new signals, unless there's a strong reason not to (e.g. values aren't comparable between geos). Is this handled elsewhere in the code?

Reply:
https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/38cd523571218ec2405c79cec0fc6b427ee86de8